sympy/geometry/line.py

```diff
@@ -1083,24 +1093,68 @@ class Line(LinearEntity):
     >>> s = Segment((0, 0), (0, 1))
     >>> Line(s).equation()
     x
+
+    >>> # a circle object is returned
+    >>> from sympy.abc import x, y, a, b
+    >>> Line(3 * x + y + 18)
+    Line2D(Point2D(0, 0), Point2D(1, -3))
```

Comment:
`(0,0)` is not on the line described by `3x+y+18`

Reply:
@smichr  Yes, that was a mistake. It has been corrected now with the new approach. Please take a look.